### PR TITLE
Feature/lap 51 show name of floors

### PR DESCRIPTION
**Pull request by lapisoft**

### CODE REVIEW

File: SitumWayfinding/Classes/Positioning/Internal/SITFloorExtension.swift [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+import Foundation
+import SitumSDK
+
+extension SITFloor {
+    /**
+     Description of floor, if the floor has a name show this name otherwise show the Numeric value representating
+     the ground level.
+    */
+    open override var description: String {
```
Comment (lapisoft) [inline SitumWayfinding/Classes/Positioning/Internal/SITFloorExtension.swift:13]: Change name to floorUILabel

File: SitumWayfinding/Classes/Positioning/Internal/SITBuildingExtension.swift [hunk 2]
```diff
@@ -0,0 +1,12 @@
+//
+// Created by Lapisoft MacPro on 19/12/22.
+//
+
+import Foundation
+import SitumSDK
+
+extension SITBuildingInfo {
+    func buildingFloorDescription(_ floor: SITFloor) -> String {
```
Comment (lapisoft) [inline SitumWayfinding/Classes/Positioning/Internal/SITBuildingExtension.swift:9]: buildingFloorUILabel

File: SitumWayfinding/Classes/Positioning/PositioningViewController.swift [hunk 3]
```diff
@@ -1239,9 +1240,17 @@ extension PositioningViewController {
...
+            if let floor = poi.floor(activeBuildingInfo: buildingInfo) {
+                secondaryLabel = buildingInfo.buildingFloorDescription(floor)
+            } else {
+                secondaryLabel = self.buildingName
+            }
+        }
         self.updateInfoBarLabelsIfNotInsideRoute(
             mainLabel: poiMarker.poi?.name ?? DEFAULT_POI_NAME,
-            secondaryLabel: self.buildingName
+            secondaryLabel: secondaryLabel
```
Comment (lapisoft) [inline SitumWayfinding/Classes/Positioning/PositioningViewController.swift:1253]: Try to make a method to retrieve the floor an unify with containerInfoBarMap?.setLabels

File: SitumWayfinding/Classes/Positioning/Search/SearchResultsTableViewController.swift [hunk 4]
```diff
@@ -11,7 +11,7 @@ protocol SearcheableItem {
     var id: String { get }
     var name: String { get }
     //TODO make it more generic
-    func floor(activeBuildingInfo: SITBuildingInfo?) -> String
+    func floorDescription(activeBuildingInfo: SITBuildingInfo?) -> String
```
Comment (lapisoft) [inline SitumWayfinding/Classes/Positioning/Search/SearchResultsTableViewController.swift:14]: change to floorUILabel

**Review by fsvilas — APPROVED** [review]

**Pull request merged**